[IOS] Admob - You attempted to use a firebase module that's not installed natively on your iOS project by calling firebase.admob()

<!---
Hello there you awesome person;
Please note that the issue list of this repo is exclusively for bug reports;

1) For feature requests please visit our [Feature Request Board](https://boards.invertase.io/react-native-firebase).
2) For questions and support please use our Discord chat: https://discord.gg/C9aK28N or Stack Overflow: https://stackoverflow.com/questions/tagged/react-native-firebase
3) If this is a setup issue then please make sure you've correctly followed the setup guides, most setup issues such as 'duplicate dex files', 'default app has not been initialized' etc are all down to an incorrect setup as the guides haven't been correctly followed.
-->

<!-- NOTE: You can change any of the `[ ]` to `[x]` to mark an option(s) as selected -->

<!-- PLEASE DO NOT REMOVE ANY SECTIONS FROM THIS ISSUE TEMPLATE   -->
<!--   Leave them as they are even if they're irrelevant to your issue -->

## Issue

<!-- Please describe your issue here --^ and provide as much detail as you can. -->
<!-- Include code snippets that show your usages of the library in the context of your project. -->
<!-- Snippets that also show how and where the library is imported in JS are useful to debug issues relating to importing or methods not found issues -->


I am having an issue on iOS with the adMob pod.
On Android everything works fine.
On iOS I get the following error when trying to call firebase.admob() :

```
Error: You attempted to use a firebase module that's not installed natively on your iOS project by calling firebase.admob().

Ensure you have the required Firebase iOS SDK pod for this module included in your Podfile, in this instance confirm you've added "pod 'Firebase/AdMob'" to your Podfile

See http://invertase.link/ios for full setup instructions.

This error is located at:
    in App (at renderApplication.js:34)
    in RCTView (at View.js:45)
    in View (at AppContainer.js:98)
    in RCTView (at View.js:45)
    in View (at AppContainer.js:115)
    in AppContainer (at renderApplication.js:33)

initialiseNativeModule
    native.js:43:20
ModuleBase
    ModuleBase.js:26:27
AdMob
    index.js:21:4
<unknown>
    apps.js:66:74
getModule
    apps.js:186:20
App
    App.js:98:21
App
    [native code]:0
<unknown>
    createClassProxy.js:98:23
instantiate
    createClassProxy.js:96:6
constructClassInstance
    ReactNativeRenderer-dev.js:8864:26
updateClassComponent
    ReactNativeRenderer-dev.js:11508:6
performUnitOfWork
    ReactNativeRenderer-dev.js:16075:21
workLoop
    ReactNativeRenderer-dev.js:16115:41
renderRoot
    ReactNativeRenderer-dev.js:16219:15
performWorkOnRoot
    ReactNativeRenderer-dev.js:17198:17
performWork
    ReactNativeRenderer-dev.js:17099:24
performSyncWork
    ReactNativeRenderer-dev.js:17060:14
requestWork
    ReactNativeRenderer-dev.js:16925:19
scheduleWork
    ReactNativeRenderer-dev.js:16724:16
scheduleRootUpdate
    ReactNativeRenderer-dev.js:17413:15
render
    ReactNativeRenderer-dev.js:18191:20
renderApplication
    renderApplication.js:59:34
run
    AppRegistry.js:101:10
runApplication
    AppRegistry.js:195:26
__callFunction
    MessageQueue.js:366:47
<unknown>
    MessageQueue.js:106:26
__guard
    MessageQueue.js:314:10
callFunctionReturnFlushedQueue
    MessageQueue.js:105:17
callFunctionReturnFlushedQueue
    [native code]:0

```

I've tried re-do the initial setup guide on my project. And adding this pod:
`pod 'RNFirebase', :path => '../node_modules/react-native-firebase/ios'`

but I still have this issue

---

## Project Files

<!-- Provide the contents of key project files which will help to debug -->
<!--     For Example: -->
<!--        - iOS: `Podfile` contents. -->
<!--        - Android: `android/build.gradle` contents. -->
<!--        - Android: `android/app/build.gradle` contents. -->
<!--        - Android: `AndroidManifest.xml` contents. -->

<!-- ADD THE CONTENTS OF THE FILES IN THE PROVIDED CODE BLOCKS BELOW -->

### iOS

<details><summary>Click To Expand</summary>
<p>

#### `ios/Podfile`:

- [ ] I'm not using Pods
- [x] I'm using Pods and my Podfile looks like:

```
platform :ios, '9.0'

target 'xxx' do
  # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
  # use_frameworks!

  # Pods for xxx
  pod 'React', :path => '../node_modules/react-native'
  pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
  pod 'Firebase/Core', '~> 5.15.0'
  pod 'Firebase/Database', '~> 5.15.0'
  pod 'Firebase/AdMob', '~> 5.15.0'
  pod 'RNCAsyncStorage', :path => '../node_modules/@react-native-community/async-storage'

  target 'xxx' do
    inherit! :search_paths
    # Pods for testing    

  end

  target 'xxx' do
    inherit! :search_paths
    # Pods for testing
  end

end

```
`podfile.lock`:

```
PODS:
  - Firebase/AdMob (5.15.0):
    - Firebase/Core
    - Google-Mobile-Ads-SDK (~> 7.37)
  - Firebase/Core (5.15.0):
    - Firebase/CoreOnly
    - FirebaseAnalytics (= 5.4.0)
  - Firebase/CoreOnly (5.15.0):
    - FirebaseCore (= 5.1.10)
  - Firebase/Database (5.15.0):
    - Firebase/CoreOnly
    - FirebaseDatabase (= 5.0.4)
  - FirebaseAnalytics (5.4.0):
    - FirebaseCore (~> 5.1)
    - FirebaseInstanceID (~> 3.3)
    - GoogleAppMeasurement (= 5.4.0)
    - GoogleUtilities/AppDelegateSwizzler (~> 5.2)
    - GoogleUtilities/MethodSwizzler (~> 5.2)
    - GoogleUtilities/Network (~> 5.2)
    - "GoogleUtilities/NSData+zlib (~> 5.2)"
    - nanopb (~> 0.3)
  - FirebaseAuthInterop (1.0.0)
  - FirebaseCore (5.1.10):
    - GoogleUtilities/Logger (~> 5.2)
  - FirebaseDatabase (5.0.4):
    - FirebaseAuthInterop (~> 1.0)
    - FirebaseCore (~> 5.1)
    - leveldb-library (~> 1.18)
  - FirebaseInstanceID (3.3.0):
    - FirebaseCore (~> 5.1)
    - GoogleUtilities/Environment (~> 5.3)
    - GoogleUtilities/UserDefaults (~> 5.3)
  - Google-Mobile-Ads-SDK (7.41.0)
  - GoogleAppMeasurement (5.4.0):
    - GoogleUtilities/AppDelegateSwizzler (~> 5.2)
    - GoogleUtilities/MethodSwizzler (~> 5.2)
    - GoogleUtilities/Network (~> 5.2)
    - "GoogleUtilities/NSData+zlib (~> 5.2)"
    - nanopb (~> 0.3)
  - GoogleUtilities/AppDelegateSwizzler (5.6.0):
    - GoogleUtilities/Environment
    - GoogleUtilities/Logger
    - GoogleUtilities/Network
  - GoogleUtilities/Environment (5.6.0)
  - GoogleUtilities/Logger (5.6.0):
    - GoogleUtilities/Environment
  - GoogleUtilities/MethodSwizzler (5.6.0):
    - GoogleUtilities/Logger
  - GoogleUtilities/Network (5.6.0):
    - GoogleUtilities/Logger
    - "GoogleUtilities/NSData+zlib"
    - GoogleUtilities/Reachability
  - "GoogleUtilities/NSData+zlib (5.6.0)"
  - GoogleUtilities/Reachability (5.6.0):
    - GoogleUtilities/Logger
  - GoogleUtilities/UserDefaults (5.6.0):
    - GoogleUtilities/Logger
  - leveldb-library (1.20)
  - nanopb (0.3.901):
    - nanopb/decode (= 0.3.901)
    - nanopb/encode (= 0.3.901)
  - nanopb/decode (0.3.901)
  - nanopb/encode (0.3.901)
  - React (0.58.6):
    - React/Core (= 0.58.6)
  - React/Core (0.58.6):
    - yoga (= 0.58.6.React)
  - RNCAsyncStorage (1.4.0):
    - React
  - yoga (0.58.6.React)

DEPENDENCIES:
  - Firebase/AdMob (~> 5.15.0)
  - Firebase/Core (~> 5.15.0)
  - Firebase/Database (~> 5.15.0)
  - React (from `../node_modules/react-native`)
  - "RNCAsyncStorage (from `../node_modules/@react-native-community/async-storage`)"
  - yoga (from `../node_modules/react-native/ReactCommon/yoga`)

SPEC REPOS:
  https://github.com/cocoapods/specs.git:
    - Firebase
    - FirebaseAnalytics
    - FirebaseAuthInterop
    - FirebaseCore
    - FirebaseDatabase
    - FirebaseInstanceID
    - Google-Mobile-Ads-SDK
    - GoogleAppMeasurement
    - GoogleUtilities
    - leveldb-library
    - nanopb

EXTERNAL SOURCES:
  React:
    :path: "../node_modules/react-native"
  RNCAsyncStorage:
    :path: "../node_modules/@react-native-community/async-storage"
  yoga:
    :path: "../node_modules/react-native/ReactCommon/yoga"

SPEC CHECKSUMS:
  Firebase: 8bb9268bff82374f2cbaaabb143e725743c316ae
  FirebaseAnalytics: c06f9d70577d79074214700a71fd5d39de5550fb
  FirebaseAuthInterop: 0ffa57668be100582bb7643d4fcb7615496c41fc
  FirebaseCore: 35747502d9e8c6ee217385ad04446c7c2aaf9c5c
  FirebaseDatabase: 0621689f77528d62b47e1c06ca737c4c19275d1a
  FirebaseInstanceID: e2fa4cb35ef5558c200f7f0ad8a53e212215f93e
  Google-Mobile-Ads-SDK: 101ce89b94fdf60ec64ba7b6e9b9d812dc765936
  GoogleAppMeasurement: 98b71f5e04142793729a5ef23e5b96651ff4b70f
  GoogleUtilities: f44b068df02195da10409f415bd8c98439cd70fa
  leveldb-library: 08cba283675b7ed2d99629a4bc5fd052cd2bb6a5
  nanopb: 2901f78ea1b7b4015c860c2fdd1ea2fee1a18d48
  React: 130b87b2d5e2baac646954282cab87be986d98fc
  RNCAsyncStorage: b82dc6e5b39a625d70e3b3492bff75c0de94ba71
  yoga: 32d7ef1081951e9a35a4c72a7be797598b138a48

PODFILE CHECKSUM: 7c7dc6ad557e143e89cd532086f991b949c19580

COCOAPODS: 1.6.1

```

#### `AppDelegate.m`:

```objc
// N/A
```

</p>
</details>

---

### Android

<details><summary>Click To Expand</summary>
<p>

#### `android/build.gradle`:

```groovy
// N/A
```

#### `android/app/build.gradle`:

```groovy
// N/A
```

#### `android/settings.gradle`:

```groovy
// N/A
```

#### `MainApplication.java`:

```java
// N/A
```

#### `AndroidManifest.xml`:

```xml
<!-- N/A -->
```

</p>
</details>


---

## Environment

<details><summary>Click To Expand</summary>
<p>

**`react-native info` output:**

<!-- Please run `react-native info` on your terminal and paste the contents into the code block below -->

```
 React Native Environment Info:
    System:
      OS: macOS 10.14.2
      CPU: (4) x64 Intel(R) Core(TM) i5-4278U CPU @ 2.60GHz
      Memory: 1.17 GB / 8.00 GB
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 11.6.0 - /usr/local/bin/node
      Yarn: 1.12.3 - /usr/local/bin/yarn
      npm: 6.5.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
      Android SDK:
        API Levels: 27, 28
        Build Tools: 27.0.3, 28.0.3
        System Images: android-18 | Google APIs Intel x86 Atom, android-19 | Google APIs Intel x86 Atom, android-22 | Google APIs Intel x86 Atom, android-26 | Google APIs Intel x86 Atom, android-28 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 3.2 AI-181.5540.7.32.5056338
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.8.3 => 16.8.3 
      react-native: 0.58.6 => 0.58.6 
    npmGlobalPackages:
      react-native-app-id: 0.0.5
      react-native-cli: 2.0.1
```

<!-- change `[ ]` to `[x]` to select an option(s) -->

- **Platform that you're experiencing the issue on**:
  - [x] iOS
  - [ ] Android
  - [ ] **iOS** but have not tested behavior on Android
  - [ ] **Android** but have not tested behavior on iOS
  - [ ] Both
- **`Firebase` module(s) you're using that has the issue:**
  - `Admob`
- **Are you using `TypeScript`?**
  - `N`
  
</p>
</details>


<!-- Thanks for reading this far down ❤️  -->
<!-- High quality, detailed issues are much easier to triage for maintainers -->

<!-- For bonus points, if you put a 🔥 (:fire:) emojii at the start of the issue title we'll know -->
<!-- that you took the time to fill this out correctly, or, at least read this far -->

---

Think `react-native-firebase` is great? Please consider supporting all of the project maintainers and contributors by donating via our [Open Collective](https://opencollective.com/react-native-firebase/donate) where all contributors can submit expenses. [[Learn More]](https://invertase.io/oss/react-native-firebase/contributing/donations-expenses)

- 👉 Check out [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter for updates on the library.

I've tried out the demo twice, but it keeps showing me black screen then it closes itself. 

Also, I forgot to say, every time I tried to run **my app**, I keep getting _Build Failed_ with message

```
error: PCH was compiled with module cache path '/Users/KuoyHuot/Desktop/React Native/xxx/ios/build/ModuleCache.noindex/147FY0L6K85ZP', but the path is currently '/Users/KuoyHuot/Desktop/React Native/Working/xxx/ios/build/ModuleCache.noindex/147FY0L6K85ZP'
1 error generated.
While building module 'UIKit' imported from /Users/KuoyHuot/Desktop/React Native/Working/xxx/node_modules/react-native/React/Views/UIView+React.h:8:
While building module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator12.1.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
In file included from <module-includes>:1:
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator12.1.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:10: fatal error: could not build module 'CoreFoundation'
#include <CoreFoundation/CoreFoundation.h>
 ~~~~~~~~^
2 errors generated.
While building module 'UIKit' imported from /Users/KuoyHuot/Desktop/React Native/Working/xxx/node_modules/react-native/React/Views/UIView+React.h:8:
In file included from <module-includes>:1:
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator12.1.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:9: fatal error: could not build module 'Foundation'
#import <Foundation/Foundation.h>
 ~~~~~~~^
3 errors generated.
In file included from /Users/KuoyHuot/Desktop/React Native/Working/xxx/node_modules/react-native/React/Views/UIView+React.m:8:
/Users/KuoyHuot/Desktop/React Native/Working/xxx/node_modules/react-native/React/Views/UIView+React.h:8:9: fatal error: could not build module 'UIKit'
#import <UIKit/UIKit.h>
 ~~~~~~~^
error: PCH was compiled with module cache path '/Users/KuoyHuot/Desktop/React Native/xxx/ios/build/ModuleCache.noindex/147FY0L6K85ZP', but the path is currently '/Users/KuoyHuot/Desktop/React Native/Working/xxx/ios/build/ModuleCache.noindex/147FY0L6K85ZP'
1 error generated.
error: PCH was compiled with module cache path '/Users/KuoyHuot/Desktop/React Native/xxx/ios/build/ModuleCache.noindex/147FY0L6K85ZP', but the path is currently '/Users/KuoyHuot/Desktop/React Native/Working/xxx/ios/build/ModuleCache.noindex/147FY0L6K85ZP'
1 error generated.
6 errors generated.


** BUILD FAILED **



The following build commands failed:
	CompileC /Users/KuoyHuot/Desktop/React\ Native/Working/xxx/ios/build/Build/Intermediates.noindex/React.build/Debug-iphonesimulator/React.build/Objects-normal/x86_64/UIView+React.o /Users/KuoyHuot/Desktop/React\ Native/Working/xxx/node_modules/react-native/React/Views/UIView+React.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
```
But my app still run fine. Could this be the cause?

--- a/ios/RNFirebase.xcodeproj/project.pbxproj
+++ b/ios/RNFirebase.xcodeproj/project.pbxproj
@@ -515,6 +515,7 @@
 					"$(SRCROOT)/../../../ios/Pods/FirebaseRemoteConfig/Frameworks",
 					"$(SRCROOT)/../../../ios/Pods/FirebaseStorage/Frameworks",
 					"$(SRCROOT)/../../../ios/Pods/Google-Mobile-Ads-SDK/Frameworks/frameworks",
+					"$(SRCROOT)/../../../ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current",
 				);
 				HEADER_SEARCH_PATHS = (
 					"$(inherited)",

